packages/frontend/src/components/Search/index.tsx

```diff
@@ -41,6 +41,12 @@ export const Search = ({ isDesktop = false }: SearchProps) => {
     }
```

Comment:
`enterSearch` and `search` functions have duplicate code. Could you place `search()` in `enterSearch`?
```
const enterSearch = (event: React.KeyboardEvent) => {
    if (event.code === 'Enter' && query) {
      search();
    }
```